cfme/tests/cli/test_appliance_console_db_restore.py

```diff
@@ -207,11 +205,22 @@ def fetch_v2key(appl1, appl2):
     appl2.ssh_client.put_file(rand_yml_filename, "/var/www/miq/vmdb/config/database.yml")
 
 
-def fetch_db_local(appl1, appl2):
+def fetch_db_local(appl1, appl2, file_name):
     # Fetch db from the first appliance
     dump_filename = "/tmp/db_dump_{}".format(fauxfactory.gen_alphanumeric())
-    appl1.ssh_client.get_file("/tmp/evm_db.backup", dump_filename)
-    appl2.ssh_client.put_file(dump_filename, "/tmp/evm_db.backup")
+    appl1.ssh_client.get_file(file_name, dump_filename)
+    appl2.ssh_client.put_file(dump_filename, file_name)
+
+
+@pytest.fixture
+def two_appliances_one_with_providers(temp_appliances_preconfig_funcscope):
+    """Requests two configured appliances from sprout."""
+    appl1, appl2 = temp_appliances_preconfig_funcscope
+
+    # Add infra/cloud providers
+    provider_app_crud(VMwareProvider, appl1).setup()
+    provider_app_crud(EC2Provider, appl1).setup()
+    return appl1, appl2
 
 
 def setup_nfs_samba_backup(appl1):
```

Comment:
This method and all the references, as well as the comments will be removed in the followup PR